### ISSUE TITLE
Accuracy problem on CIFAR100 dataset

### ISSUE THREAD

**Issue by NikosKats** [AminParvaneh/alpha_mix_active_learning#7]
I got the mentioned results for the dataset MNIST, but when I use the CIFAR100 dataset with these parameters:

--data_name CIFAR100 --data_dir your_data_directory --log_dir your_log_directory \
 --n_init_lb 1000 --n_query 1000 --n_round 10 --learning_rate 0.001 --n_epoch 2000 --seeds
1 --model vit_small  --pretrained_model true --fine_tune_layers 1 \
 --strategy AlphaMixSampling --alpha_opt

I get maximum accuracy of 0.3843 instead of 52.5 as mentioned in the paper. Can you please clarify if I use the correct parameters or not?
